web-components/src/components/molecules/PrevNextButtons/PrevNextButtons.tsx

```diff
@@ -9,36 +9,44 @@ type Props = {
   onSave?: () => void;
   saveDisabled: boolean;
   saveText: string;
+  className?: string;
 };
 export const PrevNextButtons = ({
   onPrev,
   onSave,
   saveDisabled,
   saveText,
+  className,
 }: Props) => {
   const { classes } = useStyles();
   return (
-    <>
+    <div className={className}>
       {onPrev && (
         <OutlinedButton
-          className={cn(classes.btn, 'mr-10 sm:mr-20')}
+          className={cn(
+            classes.btn,
+            'mr-10 sm:mr-20 h-50 sm:h-60 w-50 sm:w-60',
+          )}
           onClick={onPrev}
         >
           <ArrowDownIcon
             fontSize="small"
             direction="prev"
-            // color={theme.palette.secondary.main}
+            className="w-[24px] h-[24px]"
           />
         </OutlinedButton>
       )}
       <ContainedButton
         type="submit"
-        className={classes.btn}
+        className={cn(
+          classes.btn,
+          '!py-[13px] sm:!py-[17px] !px-[30px] sm:!px-[43px] h-50 sm:h-60',
```

Comment:
should we rework `classes.btn` (which has some padding defined that we overwrite here) or just remove it from this className and use tailwind classes entirely, instead of using `!important` here?